relay-metrics/src/aggregation.rs

```diff
@@ -787,7 +788,12 @@ impl Bucket {
 
         // Perform a split with the remaining space after adding the key. We assume an average
         // length of 8 bytes per value and compute the number of items fitting into the left side.
-        let split_at = (size - own_size) / AVG_VALUE_SIZE;
+        let split_at = max(1, (size - own_size) / AVG_VALUE_SIZE);
```

Comment:
A `split_at` value of `0` could result in empty buckets, causing an infinite loop.